pkg/systemd/logs.js

```diff
@@ -196,6 +202,22 @@ $(function() {
             }
         }
 
+        function append_service_menu(service){
+            if(!services_list.includes(service)){
+                services_list.push(service);
+                
+                //add service to the menu list and sort
+                $('#journal-service-lists').append('<li><a data-service="' + service + '">' + service + '</a></li>');
```

Comment:
Would be better not to do string interpolation with variables. But instead add each element directly and use text() and attr() functions to set values that come from variables. Avoids injection bugs.